README.md

```diff
@@ -227,6 +227,36 @@ Software is installed in miniconda environments.  You can use "[conda list](http
 ### How can I contribute to existing Open-Source Fornax notebook tutorials?
   * open issue or PR on Fornax Github [repo](https://github.com/nasa-fornax/fornax-demo-notebooks)
 
+## Using Fornax Software Environments Outside Fornax
+The Fornax software environments are available as docker container images. To use them on your computer, you will need to install
+[docker](https://docs.docker.com/engine/install/). The following are the steps needed to run the notebooks locally using the Fornax images:
```

Comment:
I did this years ago and it was a somewhat involved process.  If this is still the case, maybe a warning is in order for users so that they know this is not some simple 'pip install Docker'.  Maybe something like " You can expect this will take ~N - N  minutes and N Mb of space on your personal computer"